### PR TITLE
Remove duplicate quotes

### LINKED ISSUE #1361: Removing quote duplicates [low priority] [good first issue] [improvement]
I have noticed that there are several duplicate quotes. If no one is busy, this could be something that could be worked on.

### PR DESCRIPTION
**Pull request by omackenzie**
Fixes #1361 
I wrote a quick Python script to remove any duplicate quotes. It found and removed 1 in code_rust, 4 in english and 2 in turkish.

### CODE REVIEW

**Comment by omackenzie** [comment]
IDs of duplicate quotes are:

In code_rust.json:
- 50 and 52

In english.json:
- 4963 and 5448
- 5032 and 5450
- 261 and 5470
- 5160 and 5474

In turkish.json
- 8 and 12
- 15 and 19

File: static/quotes/code_rust.json [hunk 1]
```diff
@@ -1,1673 +1,1667 @@
...
+      "id": 34,
+      "length": 28,
+      "source": "Convert real number to string with 2 decimal places - programming-idioms.org",
+      "text": "let s = format!(\"{:.2}\", x);"
+    },
+    {
+      "id": 35,
+      "length": 13,
+      "source": "Assign to string the japanese word ネコ - programming-idioms.org",
+      "text": "let s = \"ネコ\";"
```
Comment (Miodec) [inline static/quotes/code_rust.json:218]: Please revert this change

File: static/quotes/code_rust.json [hunk 2]
```diff
@@ -1,1673 +1,1667 @@
...
+      "id": 51,
+      "length": 100,
+      "source": "First-class function : compose - programming-idioms.org",
+      "text": "fn compose<A, B, C>(f: impl Fn(A) -> B, g: impl Fn(B) -> C) -> impl Fn(A) -> C {\n\tmove |x| g(f(x))\n}"
+    },
+    {
+      "id": 52,
+      "length": 100,
+      "source": "First-class function : generic composition - programming-idioms.org",
+      "text": "fn compose<A, B, C>(f: impl Fn(A) -> B, g: impl Fn(B) -> C) -> impl Fn(A) -> C {\n\tmove |x| g(f(x))\n}"
```
Comment (Miodec) [inline static/quotes/code_rust.json:320]: As far as i can tell this was not a duplicate

**Comment by omackenzie** [comment]
I think I've fixed the issues. As for the rust quote, yes they technically have different titles, but the text is the exact same. I reverted the change but I think it's something to look at.

**Comment by Miodec** [comment]
The pr is still full of whitespace changes 
image

**Comment by omackenzie** [comment]
Should be fixed now. My script changed the line endings from LF to CRLF.

**Comment by Miodec** [comment]
Nice.
